### ISSUE TITLE
Desktop asking to set up Git LFS for Unity projects

### ISSUE THREAD

**Issue by AShenawy** [desktop/desktop#19355]
### The problem

When creating a new repository (and setting .gitignore to Unity from the list) for a new empty Unity project, GitHub Desktop requests setting up Git LFS for the project.
If the .gitignore is already present in the root folder before setting up the repository, then Desktop will skip LFS and not ask the user to set it up.

Generally, the LFS request shouldn't trigger for an empty project with a proper .gitignore, because non of the files exceed 100MB.

### Release version

3.4.5 (x64)

### Operating system

Windows 10 Pro

### Steps to reproduce the behavior

1. Create a new empty Unity project
2. Create a new repository for the project in GitHub Desktop
3. Fill in all the required details (repo name, path, and .gitignore)
4. When the repo is being set up, the Git LFS window pops up

Steps to create a repository without the Git LFS pop up
1. Create a new empty Unity project
2. Place a Unity .gitignore file in the project's root folder
3. Create a new repository for the project in GitHub Desktop
4. Fill in the details (selecting the .gitignore here is irrelevant) and click "create repository"
5. The repo is set up, and no Git LFS setup pop up shows up

### Screenshots

![Desktop_LFS](https://github.com/user-attachments/assets/33ac2e15-a208-4893-bca6-fcae6e59e5cf)

**Comment by steveward** [comment]
@AShenawy thanks for the issue. This is being picked up by the `.gitattributes` file in your repository -- I would check to see if that's being created when you create the project in Unity. GitHub Desktop does a check to see if there are any `.gitattributes` entires that relate to Git LFS, and then prompts for LFS to be initialized if those are detected. 

I hope that helps explain things. Let me know if you have any other questions.

**Comment by AShenawy** [comment]
Hey @steveward 
Thank you for the quick answer and checking this issue.
Unity doesn't create any `.gitattribute` files when creating a new project. These are all created by GitHub Desktop. Also, the steps I mentioned to recreate both cases can be done over and over on the same project (which I did) by removing the Git files (`.gitattributes`, `.gitignore`, and the `.git` folder), clearing the repo, and creating it anew. I've managed to trigger the LFS popup and not trigger it several times in a row, depending on whether the `.gitignore` file existed or not before creating the repo in GitHub Desktop.

I think it might be worth saying that this occurrence is new-ish with Desktop. I've been using it for years now with Unity and I've never had the LFS initialisation popup unless the `.gitignore` is missing or set up wrongfully. A fresh new Unity project doesn't need LFS to be set up.

Hope that clarifies the issue.

Update: I've attached an image of the contents of a new Unity project, fresh out of the oven, untouched by Git.
![Unity Project](https://github.com/user-attachments/assets/70c5c847-2a4d-4d81-8c76-255f964ba03b)

**Comment by steveward** [comment]
@AShenawy I just tested this out and was unable to reproduce it when creating a new project in Unity. Can you run `git lfs ls-files` on one of these projects from the command line and see what the output shows? That should show any files that are being tracked by Git LFS, and may help narrow down the issue

Is there any chance you have a global `.gitattributes` file that could be causing this?

**Comment by AShenawy** [comment]
@steveward sorry for the late reply.
I don't have a global `.gitattributes` file. Now that I read that you couldn't reproduce it, I've tested it with 3 different versions of Unity: 2020.3.42, 2021.3.35, and 2022.3.33

Surprisingly, I was able to reproduce the issue every time on both 2020.3 and 2021.3. However, now on the 2022.3 Unity version I didn't get the Git LFS prompt when setting up the repo. I also tried it twice, once where the `.gitignore` is added using GitHub Desktop on the create repo window, and another with it being already in the project folder beforehand. Both times I wasn't asked to setup Git LFS.

As for checking the output from running `git lfs ls-files` well, I got nothing. It was empty, on both 2020.3 and 2021.3 where it required that I set it up during the repo creation.
